pandas/core/frame.py

```diff
@@ -1964,6 +1966,40 @@ def to_feather(self, path):
 
         to_feather(self, path)
 
+    def to_markdown(
+        self,
+        buf: Optional[FilePathOrBuffer[str]] = None,
+        mode: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        """
+        Print a DataFrame in markdown-friendly format.
+
+        .. versionadded:: 1.0
+
```

Comment:
Add a `Parameters` section here.

Reply:
Sure, done. Wasn't sure what to write for `mode`, as the docstring for `pandas/io/common.py::get_filepath_or_buffer` just says
```
mode : str, optional
```
and, as far as I can tell, the parameter is only used in the following conditions:
```
if is_s3_url(filepath_or_buffer):
if is_gcs_url(filepath_or_buffer):
```